### ISSUE TITLE
EMR Docs

### ISSUE THREAD

**Issue by mbossenbroek** [Netflix/PigPen#53]
Add docs for how to run e2e on EMR w/ s3

**Comment by technion** [comment]
@mbossenbroek I'm interested in doing this - does PigPen introduce any complications beyond AWS' documented "submit a pig script" process?

If I can get enough hints to make it work I'd be happy to write a step by step for this issue.

**Comment by mbossenbroek** [comment]
I don't believe it should be any different. You should be able to build an uberjar, generate a script, and run it. Looking forward to see what you come up with!

**Comment by technion** [comment]
Hi @mbossenbroek,

Unfortunately I can't file a PR against the wiki, so I'm unsure how you'd like me to best supply these docs.

I've documented PigPen in an EMR environment for our use here, the guide is completely generic and you're free to use it:

https://github.com/technion/bigdatawanna/tree/master/pigpen

**Comment by mbossenbroek** [comment]
Thanks! I'll incorporate this into the wiki
